### PR TITLE
libomxil-bellagio: update formula

### PR DESCRIPTION
**Pull request by maxim-belkin**
update formula

### CODE REVIEW

**Comment by maxim-belkin** [comment]
@psubnwell @Gwang-Jin 
Could you please test this PR using the steps below and report your gist-logs?

```sh
cd $(brew --repo linuxbrew/xorg)
git pull origin master
git fetch origin pull/445/head:pr445
git checkout pr445
brew install -vds libomxil-bellagio
brew gist-logs libomxil-bellagio
brew remove libomxil-bellagio
brew install -vds --with-docs libomxil-bellagio
brew gist-logs libomxil-bellagio
```

**Comment by maxim-belkin** [comment]
https://gist.github.com/1430889334658ba85c76944d26234c3d

**Comment by maxim-belkin** [comment]
@iMichka and @sjackman 
Could you please test this formula?

**Comment by Gwang-Jin** [comment]
Sorry, now my linuxbrew was broken
```
configure: error: C compiler cannot create executables
```

**Comment by maxim-belkin** [comment]
That's... not caused by this change, right? Please post corresponding gist-logs link

File: Formula/libomxil-bellagio.rb [hunk 1]
```diff
@@ -12,27 +12,35 @@ class LibomxilBellagio < Formula
...
       --localstatedir=#{var}
       --disable-dependency-tracking
+      --enable-android=no
+      --enable-audioeffects
+      --enable-clocksrc
+      --enable-components
+      --enable-debug=no
+      --enable-doc=#{build.with?("docs") ? "yes" : "no"}
+      --enable-static=#{build.with?("static") ? "yes" : "no"}
+      --enable-videoscheduler
```
Comment (sjackman) [inline Formula/libomxil-bellagio.rb]: I'd suggest going with the upstream defaults, unless there's a particular motivation to deviate from the defaults.

Comment (maxim-belkin) [reply Formula/libomxil-bellagio.rb]: This is, actually, the default value for `libomx...`

Comment (sjackman) [reply Formula/libomxil-bellagio.rb]: I suggest removing all the `--enable-*` options that are the default values. If they're not default, add a comment indicating why they're desirable.

Comment (maxim-belkin) [reply Formula/libomxil-bellagio.rb]: Um, I already did that. The remaining `--enable` options are there make builds reproducible.

File: Formula/libomxil-bellagio.rb [hunk 2]
```diff
@@ -12,27 +12,29 @@ class LibomxilBellagio < Formula
...
     ]
-    args << "--enable-static=#{build.with?("static") ? "yes" : "no"}"
 
     system "./configure", *args
     ENV.deparallelize
     system "make"
+    # system "make", "check" if build.with? "test"
+    # 'make check' Fails with omxvolcontroltest.h:38:22: fatal error: OMX_Core.h: No such file or directory
     system "make", "install"
+    system "make", "installcheck" if build.with? "test"
```
Comment (sjackman) [inline Formula/libomxil-bellagio.rb:36]: Nice fix for `make check`!

**Review by sjackman — APPROVED** [review]
Looks good!

File: Formula/libomxil-bellagio.rb (hunk 2 above)
Comment (maxim-belkin) [reply Formula/libomxil-bellagio.rb:36]: Thank you! :)

**Pull request merged**